CHANGELOG.md

```diff
@@ -27,6 +27,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - Support for `tr()` descriptors in the `descriptor!()` macro
 - Add support for Bitcoin Core 23.0 when using the `rpc` blockchain
 - Add `remove_partial_sigs` and `try_finalize` to `SignOptions`
+- Add `Excess` enum to handle remaining amount after coin selection.
```

Comment:
These two should probably go under "Unreleased"

Reply:
Corrected.